ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type Interface interface {
```

Comment:
OK. If it's in the standard library then I think that's a pretty strong argument. It may be possible to work around the issue of conflicting mock names. That or we can file a bug or PR for mockery.